### PR TITLE
routers: Drop LP contract call from Axelar message

### PR DESCRIPTION
**Pull request by cdamian**
# Description

Dropping the explicit call to the LP contract since our testing revealed that it is unnecessary.

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by mustermeiszer — APPROVED** [review]

**Comment by mustermeiszer** [comment]
please bumb dev runtime version to 24

**Comment by mustermeiszer** [comment]
lets merge this one into #1538 or #1538 before that.

File: pallets/liquidity-pools-gateway/routers/src/routers/axelar_evm.rs [hunk 1]
```diff
@@ -74,8 +74,8 @@ where
 }
 
 /// Encodes the provided message into the format required for submitting it
-/// to the Axelar contract which in turn submits it to the LiquidityPools
-/// contract.
+/// to the Axelar contract which in turn calls the LiquidityPools
+/// contract with the serialized LP message a payload.
```
Comment (NunoAlexandre) [inline pallets/liquidity-pools-gateway/routers/src/routers/axelar_evm.rs:78]: I think this is enough 👀 

```suggestion
/// contract with the serialized LP message.
```

File: pallets/liquidity-pools-gateway/routers/src/routers/axelar_evm.rs [hunk 2]
```diff
@@ -87,36 +87,6 @@ pub(crate) fn get_axelar_encoded_msg(
 	target_chain: Vec<u8>,
 	target_contract: H160,
 ) -> Result<Vec<u8>, &'static str> {
-	let target_chain_string =
-		String::from_utf8(target_chain).map_err(|_| "target chain conversion error")?;
-
-	#[allow(deprecated)]
```
Comment (NunoAlexandre) [inline pallets/liquidity-pools-gateway/routers/src/routers/axelar_evm.rs:93]: 🧹

**Review by NunoAlexandre — DISMISSED** [review]
LGTM, just a comment on a doc

File: pallets/liquidity-pools-gateway/routers/src/routers/axelar_evm.rs (hunk 1 above)
Comment (cdamian) [reply pallets/liquidity-pools-gateway/routers/src/routers/axelar_evm.rs:78]: Ah, that should be `with the serialized LP message as payload` ...

**Review by wischli — APPROVED** [review]

**Comment by cdamian** [comment]
Tested successfully on `moonbase-dev`.

**Review by NunoAlexandre — DISMISSED** [review]

**Review by mustermeiszer — APPROVED** [review]
LGTM

**Pull request merged**